citi-onboarding/Elementus-Jr
# Hotfix/models

**What I did**
- Adjusted map name
- Removed Example model

**How to test**
- Go to the specific registration screen and see if the name "undefined" has left the top of the page
- See if you can no longer create an instance of Example
